common/utils/convert-image-uri.js

```diff
@@ -102,7 +101,7 @@ export function convertIiifImageUri(
   originalUri: string,
   requiredSize: number | 'full'
 ): string {
-  if (determineIfGif(originalUri) || originalUri.startsWith(iiifThumbUri)) {
+  if (determineIfGif(originalUri) || !originalUri.startsWith(iiifImageUri)) {
```

Comment:
Maybe not for now, but I wonder what the gif check is for. @gestchild ?

Reply:
I wonder if it’s a Prismic thing?

I’m not sure we’d ever serve a GIF through DLCS; I kept it here because I wasn’t sure what it was for either.

Reply:
It may be a hangover from when we were serving images from wordpress